.github/workflows/create-release-pr.yml

```diff
@@ -67,4 +67,6 @@ jobs:
           - [ ] Update [CHANGELOG.md](https://github.com/apollographql/apollo-ios/blob/main/CHANGELOG.md) with all relevant changes since the prior version. _Please include PR numbers and mention contributors for external PR submissions._
 
           #### Things to do as part of releasing
-          - [ ] Update and publish the draft release created by CI using the latest [CHANGELOG.md](https://github.com/apollographql/apollo-ios/blob/main/CHANGELOG.md) contents
\ No newline at end of file
+          - [ ] Run the []'Publish Release' workflow](https://github.com/apollographql/apollo-ios-dev/actions/workflows/publish-release.yml) in apollo-ios-dev
```

Comment:
Do we need this step? The `Publish Release` workflow should run automatically when merging release PRs

Reply:
The trigger is set to `workflow_dispatch` which from the [Actions documentation indicates it's a manual trigger](https://github.com/apollographql/apollo-ios-dev/blob/main/.github/workflows/publish-release.yml#L4). Am I interpreting that incorrectly?

Reply:
Yea so that means the workflow can be manually triggered, I believe it's also needed to allow dispatching from other workflows (although i could be wrong there) but we have it being dispatched [here](https://github.com/apollographql/apollo-ios-dev/blob/main/.github/workflows/pr-subtree-push.yml#L76)

Reply:
Ah, I see. OK, I'll get another PR up to remove that step - apologies.